be/src/column/column_helper.h

```diff
@@ -166,20 +165,28 @@ class ColumnHelper {
         }
     }
 
+    // Cast to Nullable
+    static ColumnPtr cast_to_nullable_column(const ColumnPtr& src_column) {
```

Comment:
inline?

Reply:
This function is located at header file, so there's no difference of using a `inline` keyword